src/Command/Wishlist/AddWishlistProduct.php

```diff
@@ -0,0 +1,53 @@
+<?php
+
+/*
+ * This file was created by developers working at BitBag
+ * Do you need more information about us and what we do? Visit our https://bitbag.io website!
+ * We are hiring developers from all over the world. Join us and start your new, exciting adventure and become part of us: https://bitbag.io/career
+*/
+
+declare(strict_types=1);
+
+namespace BitBag\SyliusWishlistPlugin\Command\Wishlist;
+
+use BitBag\SyliusWishlistPlugin\Entity\WishlistProductInterface;
+use Sylius\Bundle\OrderBundle\Controller\AddToCartCommandInterface;
+
+class AddWishlistProduct
+{
+    private WishlistProductInterface $wishlistProduct;
```

Comment:
This is not nullable, although setter/getter allow to do this. Keep in mind that these properties won't be null, you need to set the value explicitly.